test/types/types.test.ts

```diff
@@ -48,6 +48,22 @@ Object.keys(TYPE_GENERATORS).forEach(type => {
       expect(onSave).toHaveBeenCalledWith(model);
     });
 
+    it('Has correct empty values', async () => {
+      const emptyValue = (() => {
```

Comment:
I would use a named function for this that takes a `fieldConfig` prop. I can imagine this being useful in other tests